src/base/utils/foreignapps.cpp

```diff
@@ -236,6 +237,14 @@ bool Utils::ForeignApps::PythonInfo::isValid() const
     return (!executableName.isEmpty() && version.isValid());
 }
 
+bool Utils::ForeignApps::PythonInfo::isSupportedVersion() const
+{
+    const int majorVer = version.majorNumber();
+    return ((majorVer > 3)
```

Comment:
Would be nice to move these numbers to a data file or build system rule, maybe...

Reply:
I would defer it later if it is not critical for fixing the regression.